sandbox/settings.py

```diff
@@ -305,8 +310,7 @@
 
 # Taken from PayPal's documentation - these should always work in the sandbox
 PAYPAL_SANDBOX_MODE = True
-PAYPAL_CALLBACK_HTTPS = False
```

Comment:
Let's leave it for now, for backwards compatibility and replace in 3.0 (alongside with other breaking changes).